src/app/components/shell/router-helpers/fallback-to.js

```diff
@@ -29,16 +23,14 @@ export default function FallbackTo({name}) {
     // i think page-data-utils:fetchFromCMS would have to be updated
     // to do something special on a 404 status
     if ('error' in data) {
-        return <Layout name="default"><Error404 /></Layout>;
+        return <Error404 />;
     }
 
     if (['pages.FlexPage', 'pages.RootPage'].includes(data.meta.type)) {
-        return <Layout data={data} name={data.layout[0]?.type || 'default'}>
-            <FlexPage data={data} />
-        </Layout>;
+        setLayoutParameters({data, name: data.layout[0]?.type || 'default'});
```

Comment:
i think its usually considered to be a no-no to call a state setter in a render function, i think you're supposed to put it in a `useEffect` handler so it only calls when the inputs change?